package.json

```
@@ -27,7 +27,8 @@
         "lint": "eslint src/ tests/",
         "format": "prettier --write '**/*.{js,json,md,html,css}'",
         "format:check": "prettier --check '**/*.{js,json,md,html,css}'",
-        "release": "npm publish --access public",
+        "release": "yarn release:publish",
+        "release:publish": "npm publish --access public",
```

```suggestion
        "release:publish": "npm publish --access public --registry https://registry.npmjs.org",
```

May as well be explicit, since otherwise you might have a different registry in your `~/.npmrc` when you publish.